### PR TITLE
Cmdct 3865 - ILOS content changes

### PR DESCRIPTION
**Pull request by britt-mo**
### Description
Content changes for ILOS in Section and D, see [Figma ](https://www.figma.com/design/GBktI8ZIJM9YYcdNva0ltg/MDCT---MCR?node-id=11039-58568&t=rH4sVBPcgUBMl1pE-1)for changes (in magenta)

### Related ticket(s)

CMDCT-3865

---
### How to test
Sign in as state user
go to `<AddEditReportModal />` change line 44 to `true`
Easiest way to check the changes: 
Go directly to section D ILOS and see that the `alert` is visible, and all text matches the spec.
Then go add a `plan`, and return to section D for ILOS and see that the `alert` is still there.
Then go back to section A and add an `ILOS` (or multiple), then go to Section D for ILOS and see that the `alert` is gone.
Enter into the drawer on that page and see that it matches the spec.
Make sure all text matches the spec.

### Notes

---
### Pre-review checklist

- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
- [ ] I have performed a self-review of my code
- [ ] I have manually tested this PR in the deployed cloud environment

---
#### Review
- [ ] Design: This work has been reviewed and approved by design, if necessary
- [ ] Product: This work has been reviewed and approved by product owner, if necessary

#### Security
_If either of the following are true, notify the team's ISSO (Information System Security Officer)._

- [ ] These changes are significant enough to require an update to the SIA.
- [ ] These changes are significant enough to require a penetration test.
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/app-api/forms/mcpar.json [hunk 1]
```diff
@@ -3955,33 +3956,33 @@
                 "children": [
                   {
                     "type": "html",
-                    "content": "This program is missing ILOS. You won't be able to complete this section until you've added all the names of approved ILOS in the program, if applicable. "
+                    "content": "This program is missing ILOSs. You won't be able to complete this section until you've added all the names of approved ILOSs for this program, if applicable. "
```
Comment (karla-vm) [inline services/app-api/forms/mcpar.json:3959]:
```suggestion
                    "content": "This program is missing ILOSs. You won't be able to complete this section until you've added all the names of authorized ILOSs for this program, if applicable. "
```

**Review by karla-vm — COMMENTED** [review]
Left one comment, looks good to me otherwise!

**Review by gmrabian — COMMENTED** [review]
Two typos (one might not be?)

side note I don't like the plural ILOSs 🤷

File: services/app-api/forms/mcpar.json [hunk 2]
```diff
@@ -3918,10 +3918,11 @@
               "section": "Section D: Plan-Level Indicators",
               "subsection": "Topic XI. ILOS",
               "spreadsheet": "D4_Plan_ILOS",
+              "alert": "<b>Beginning December 2025, this section must be completed by states that authorize ILOS. Submission of this data before December 2025 is optional.</b>",
               "info": [
                 {
                   "type": "p",
-                  "content": "If In Lieu of Services and Settings (ILOS) are available in this managed care program, report ILOS utilization for each plan during the contract rating period. Select \"Enter\" for each plan listed below to indicate whether or not ILOS were offered by the plan, select which ILOS the plan offered, and enter data on utilization for each ILOS offered."
+                  "content": "If ILOSs are authorized for this program, report for each plan: if the plan offered any ILOS; if “Yes”, which ILOS the plan offered; and utilization data for each ILOS offered. If the plan offered an ILOS during the contract rating period but there was no utilization, select “Yes” that the ILOS was offered but enter “0” for utilization."
```
[the remark this answers is not in the document]
Comment (britt-mo) [reply services/app-api/forms/mcpar.json:3925]: @gmrabian line 3929 is 
`"dashboardTitle": "Report ILOS utilization for each managed care plan during the contract rating period.",`
Screenshot 2024-07-25 at 9 22 41 AM

[the remark this answers is not in the document]
Comment (karla-vm) [reply services/app-api/forms/mcpar.json:3925]: yup, confirmed on Figma 😄

**Review by karla-vm — APPROVED** [review]

File: services/app-api/forms/mcpar.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (gmrabian) [reply services/app-api/forms/mcpar.json:3925]: Oh I looks like it was included in one (no plans, no ilos mockup) but not the others. I think you're right it should not have one
Screenshot 2024-07-25 at 3 23 49 PM

**Review by gmrabian — APPROVED** [review]

**Pull request merged**